modules/oe_content_call_proposals/config/install/field.field.node.oe_call_proposals.oe_call_proposals_grants_link.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_call_proposals_grants_link
+    - node.type.oe_call_proposals
+  module:
+    - link
+id: node.oe_call_proposals.oe_call_proposals_grants_link
+field_name: oe_call_proposals_grants_link
```

Comment:
Rename this into `oe_call_proposals_grants`

Reply:
This is done.